src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -303,6 +429,58 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
                       </Tooltip>
                     )}
                   </TableCell>
+                  <TableCell
+                    sx={{
+                      px: '8px',
+                      paddingTop: '0px',
+                      paddingBottom: '0px',
+                      borderRight: '1px solid #e0e0e0',
+                    }}
+                  >
+                    {item.is_obsolete ? 'Yes' : 'No'}
+                  </TableCell>
+                  <TableCell
+                    sx={{
+                      px: '8px',
+                      paddingTop: '0px',
+                      paddingBottom: '0px',
+                      borderRight: '1px solid #e0e0e0',
+                    }}
+                  >
+                    {item.obsolete_replacement_catalogue_item_id ? (
+                      <MuiLink
+                        underline="hover"
+                        component={Link}
+                        to={`items/${item.obsolete_replacement_catalogue_item_id}`}
+                      >
+                        Click here
+                      </MuiLink>
+                    ) : (
+                      'None'
```

Comment:
Not sure this should display as None as the others don't - I don't know what it would look like in the datagrid though
![image](https://github.com/ral-facilities/inventory-management-system/assets/90245114/86551673-379b-4489-bd75-7c4effa95cd0)